vocabularies/voc4cat/0007236.ttl

```diff
@@ -5,11 +5,11 @@ PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
 <https://w3id.org/nfdi4cat/voc4cat_0007236>
     a skos:Concept ;
     dcterms:identifier "voc4cat_0007236"^^xsd:token ;
-    skos:definition "The H-Cell is a two-compartment electrochemical cell used for membrane testing, H2 permeation, or any other experiment where two separate electrode chambers are required."@en ;
+    skos:altLabel "H-cell"@en ;
+    skos:broader <https://w3id.org/nfdi4cat/voc4cat_0007250> ;
+    skos:definition "The H-Cell is a two-compartment electrochemical cell used for membrane testing, hydrogen permeation, or any other experiment where two separate electrode chambers are required."@en ;
     skos:historyNote "0009-0008-1278-8890 Michael Götte created the resource"@en ;
     skos:inScheme <https://w3id.org/nfdi4cat/voc4cat_> ;
-    skos:narrower <https://w3id.org/nfdi4cat/voc4cat_0007250> ;
     skos:prefLabel "h cell"@en ;
```

Comment:
If the term label is always written with an uppercase H it should have an uppercase H in the label: "H cell"